staff/johana-ludwig/blog/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog de idiomas y viajes</title>
+</head>
+<body>
+    <header>
+        <h1>Viaja y aprende</h1>
+        <section>
```

Comment:
In this case, the `<section>` tag is not necessary

Reply:
you are right :)